### PR TITLE
Release/74

### PR DESCRIPTION
**Pull request by alexanderdean**

### CODE REVIEW

**Comment by alexanderdean** [comment]
Hey @chuwy - this PR gives a good idea of the various things required to get a release ready...

File: 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/miscEnrichmentSpecs.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ class EtlVersionSpec extends MutSpecification {
 
   "The ETL version" should {
     "be successfully returned" in {
-      MiscEnrichments.etlVersion("hadoop-0.14.0") must_== "hadoop-0.14.0-common-0.18.0"
+      MiscEnrichments.etlVersion("hadoop-1.4.0") must_== "hadoop-1.4.0-common-0.19.0"
```
Comment (ninjabear) [inline 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/miscEnrichmentSpecs.scala]: hey @alexanderdean, this change:

`hadoop-0.14.0` -> `hadoop-1.4.0`

I'm not sure what it means? I'm just fixing a conflict for R75 and want to make sure changes like this get percolated through properly

Comment (alexanderdean) [inline 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/miscEnrichmentSpecs.scala]: In R74, we simply updates the hadoop-x.x.x to the latest available hadoop version

Comment (ninjabear) [inline 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/miscEnrichmentSpecs.scala]: Oh cool so this is an external dependency, I just need to not revert it

Comment (alexanderdean) [inline 3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/miscEnrichmentSpecs.scala]: Yep don't revert

**Pull request merged**